react/react-0.13.0.d.ts

```diff
@@ -0,0 +1,958 @@
+// Type definitions for React 0.13.0
+// Project: http://facebook.github.io/react/
+// Definitions by: Asana <https://asana.com>, AssureSign <http://www.assuresign.com>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+declare module React {
+    //
+    // React Elements 
+    // ----------------------------------------------------------------------
+    
+    type ReactType = ComponentClass<any, any, any> | string;
+
+    interface ReactElement<P> {
+        type: ReactType;
```

Comment:
This should use `ComponentClass<P, any, any> | string` instead of the type alias since type aliases don't support generics